docs/getting-started/mainnet.md

```diff
@@ -206,22 +218,22 @@ You will see JSON output where you need to locate **_catching_up_** field. When
 
 ## SECTION 3: Promote Full Node to Validator Role
 
-::: tip NOTE:
-All tasks in **SECTION 3** have to be performed as **vidulum** user created in **SECTION 1**.
-Steps in this section will allow to promote full node to validator role. If you need full node only, skip this section.
+::: warning NOTE:
+All tasks in **SECTION 3** have to be performed as the **vidulum** user created in **SECTION 1**.<br>
+**ONLY** continue this section if you have plans to impliment a validator node. 
 :::
 
-In order to create validator you need to have Vidulum account and some funds, whcih can be delegated to validator.
+In order to create a validator, you must have a Vidulum wallet and at least 1 VDL that you must self delegate as the validator.
```

Comment:
Change to at least 2 VDL to cover TX fee when creating a validator. Best practice would be at least 10 VDL but this would ultimately be up to the user

Reply:
[9e0e007](https://github.com/vidulum/documentation/pull/3/commits/9e0e0074a2c84f5b7794e96f15f9ab64c408f8d4)
Fixed.